src/index.js

```diff
@@ -1,10 +1,167 @@
 import { months } from './data.js';
 
 (async function () {
-  const allCityDetails = await fetchCityDetails()
+  let allCityDetails = await fetchCityDetails()
+  const icons = document.querySelectorAll('.mid-ic')
+  const leftArrow = document.getElementById('left-scroll-arrow')
+  const rightArrow = document.getElementById('right-scroll-arrow')
+  let middleSegmentCards = allCityDetails
+  let selectedIcon = ' '
```

Comment:
```suggestion
  let selectedWeather = ' '
```
or something similar is also fine